.gitpod.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gitpod/workspace-full
+
+# Setup NVM
+COPY .nvmrc $HOME
+RUN bash -c "source $HOME/.nvm/nvm.sh && nvm install && nvm use"
```

Comment:
Please add a new line at the end of this file, or you can use `npm run lint:fix`, it should add it by itself.